addons/@mlabs-haskell/godot-cardano/src/big_int.gd

```diff
@@ -11,6 +10,17 @@ enum Status { SUCCESS = 0, COULD_NOT_PARSE_BIGINT = 1, COULD_NOT_DESERIALIZE_BIG
 
 var _b: _BigInt
 
+@export
+var value: String:
```

Comment:
This setter is very convenient, but let's not forget to document that it can fail at runtime and that proper conversion functions should be used when creating a `BigInt` programmatically.